### ISSUE TITLE
Print the version and runner id in a banner in the log when starting a job

### ISSUE THREAD

**Issue by em-** [collabora/lava-gitlab-runner#18]
The general purpose `gitlab-runner` nicely prints this when kicking a job:

```
Running with gitlab-runner 15.7.2 (0e7679e6)
  on here-goes-the-description gPGG42Bx
```

This is extremely useful to figure out where things went wrong in case of issues or even to ensure a new version got actually picked up when deploying an update.

Can `lava-gitlab-runner`  be taught to do something similar? :)

**Comment by sjoerdsimons** [comment]
Yeah that's something that was on my mental todo list that i didn't get time to write a ticket for; fwiw this might be something to do in the runner crate rather then in the lava-gitlab-runner (so it's consistent for all runners not just hte lava one)
